### ISSUE TITLE
Create GitHub Pages for Repository

### ISSUE THREAD

**Issue by sabre1041** [openshift-mobile/openshift-mobile#69]
Create GitHub Pages for the OpenShift Mobile Application. More information can be found [here](http://pages.github.com/).

[http://openshiftmobile.com](http://openshiftmobile.com) should point to this location

**Comment by JaredBurck** [comment]
created branch for just a project site for openshift-mobile. used the automatic generator and the tactile theme for a quick start. created READ.me file for branch. added CNAME file for custom domain openshiftmobile.com. created basic content for home page and linked to demo

**Comment by JaredBurck** [comment]
site now available at http://openshiftmobile.github.io/openshift-mobile/

**Comment by JaredBurck** [comment]
next set up dns to redirect openshiftmobile.com to openshiftmobile.github.io

**Comment by sabre1041** [comment]
DNS configured to forward traffic to http://openshiftmobile.com to GitHub pages

**Comment by jyore** [comment]
Page is starting to look good. Couple notes:
- Need the logo on the page
- Once we are in the app stores, we should link to the store pages
- The demo link is useless if you do not have ripple enabled. Either we should put in a check for that on the request to the incoming link (if possible) or at least make a note that the demo requires ripple.  Third option would be to create a 'fake' service for the demo that is basically a little sandbox for users to login and play with (not hitting a real OS account)

**Comment by JaredBurck** [comment]
Finishing up a new design on the web page to give it a little more polished look. I incorporated the logo and added the app store links to the main stores. Removed the demo link all together. Began three short content sections 1) OpenShift-mobile 2) screen shots and 3) github intro. See screen capture of the new home page.

![index_osm_3](https://cloud.githubusercontent.com/assets/632932/2651725/2facf822-bf96-11e3-8b98-599295e4f6b8.PNG)
